manifests/plugin/openscap.pp

```diff
@@ -6,4 +6,10 @@
 #
 class foreman::plugin::openscap {
   foreman::plugin {'openscap': }
+
+  if $foreman::cli::manage_root_config {
```

Comment:
It'd probably have to be in its own class, yes. I don't think using `defined("foreman:cli")` would be any more reliable, it's still dependent on the parse order.